src/Val.h

```diff
@@ -1149,17 +1152,37 @@ class RecordVal final : public Val, public notifier::detail::Modifiable {
 	void Assign(int field, std::nullptr_t)
 		{ Assign(field, ValPtr{}); }
 
-	[[deprecated("Remove in v4.1.  Use GetField().")]]
-	Val* Lookup(int field) const	// Does not Ref() value.
```

Comment:
The deprecated version of this needs to remain until 4.1, since there's likely external code using it.

Reply:
Oops, good catch.  I've restored it.